### DISCUSSION TITLE
Solstice YOY (year over year) or COC (century over century)

### DISCUSSION THREAD

**Discussion by PhilfromWaterbury** [Stellarium/stellarium#3571] [Ideas]
Hi,

Looking for a script, etc, to show where on the horizon the sun first becomes visible on, for instance, the winter solstice year over year, or more specifically, century over century.
Winter solstice meaning the day when there is the least amount of daylight.  I understand the solstice to be a moment. 

Looking to visualize how the sun will track south to a certain point, then track north, then return to track south.  Does this program incorporate the effects of the moon, the sun, the tides, the lack of true roundness of the earth, Precession of the earth.

![Screenshot 2023-12-25 162638](https://github.com/Stellarium/stellarium/assets/154841805/6ef9a829-3abe-4129-b66c-a92772220261)

**Comment by gzotti** [comment]
Answered already here: https://groups.google.com/g/stellarium/c/oJUbzqSh3x8/m/5az5QudNAAAJ?utm_medium=email&utm_source=footer. Please do not post in more than 1 forum.
